wordpress-mobile/WordPress-iOS
# Followed Sites: Replace success / failure alert with Notice

Fixes #8514 

This PR replaces the alert that's displayed when you attempt to follow an already-followed site with a Notice at the bottom of the screen. It also replaces the success HUD when you follow a site.

**To test:**

* Go to Reader > Followed Sites and attempt to follow a site that you've already followed. Check that the notice is displayed at the bottom of the screen.
* On the same screen, follow a new site. Check that a different notice is displayed.
